addons/initialise-sprite-position/userscript.js

```diff
@@ -0,0 +1,22 @@
+export default async function ({ addon }) {
+  let vm = addon.tab.traps.vm;
+  let oldAddSprite = vm.addSprite;
+  
+  vm.addSprite = function (input) {
```

Comment:
probably use the prototype for 2 reasons.
1. If other addons want to pollute this too, they won't find it in the prototype
2. If you look at the VM variable in the console, you can see vm.addSprite is not behind _proto_ anymore, which we don't want to affect.

Reply:
I tried doing that but it said that vm.prototype.addSprite wasn't defined.

Reply:
It would be `vm.constructor.prototype`